### ISSUE TITLE
Incognito/cookie management for internal browser? 

### ISSUE THREAD

**Issue by retorquere** [ccrama/Slide#3012]
Slide version: 6.2
Android version: 9

Would it be possible to add cookie management or an incognito mode for the internal browser?

**Comment by OffifialMITX** [comment]
You can disable the cookies if you want to.

**Comment by retorquere** [comment]
Where do I do this?

**Comment by Tensounder54** [comment]
@OffifialMITX Are you sure this is a thing? I'm not sure where the setting for this is.

@retorquere In terms of an incognito mode, I think this is a great idea. As far as I'm aware, there is no setting for this at the moment but, this is something that we can look into.

**Comment by OffifialMITX** [comment]
Looks pretty much like a thing to me.
![](https://i.imgur.com/807TYAZ.png)

**Comment by OffifialMITX** [comment]
Android 7.1.1, though I don't think it should matter.

**Comment by Tensounder54** [comment]
Yeah that must be the setting that @OffifialMITX was talking about. I think it would be a good idea if that was mirrored in settings but I'm totally onboard with the idea of having an incognito mode. I'm not sure if this effects anything but I use mozilla-mobile/focus-android on my phone and the internal browser doesn't seem to remember my logins.

**Comment by OffifialMITX** [comment]
Probably. As far as I can gather, the browser implementation is WebView, which requires Chrome on Android 7+, but it doesn't share the settings outside of each app. And it seems that every time I re-toggk

**Comment by OffifialMITX** [comment]
Ok, no, not WebView. Disabling Chrome does not affect the app.

**Comment by Tensounder54** [comment]
I know it is WebView but I don't think it relies on Chrome

**Comment by retorquere** [comment]
I use focus too but I'd prefer to use the built in browser so I can swipe back.

**Comment by OffifialMITX** [comment]
I don't know about latter versions, but Stock 7 requires Chrome for system WebView implementation, and apps that require the system one, like Riot.im can't run without it. Though I've tested it and Slide does not seem affected. Neither it's browser.
